distributor/src/distributor/constants.bal

```diff
@@ -3,3 +3,213 @@ const ERROR_REASON = "{eclk/pubhub}Error";
 # The topic against which the publisher will publish updates and the subscribers
 # need to subscribe to, to receive result updates in `json` format.
 public const JSON_RESULTS_TOPIC = "https://github.com/ECLK/Results-Dist-json";
+
+# Map contains all the division codes
+const map<string> DIVISION_CODES = {
```

Comment:
We shouldn't put these .. the result object has all this info and taking it from outside is not good.